essay.tex

```diff
@@ -0,0 +1,64 @@
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Are User Story distribution methods used in regular Agile Development practice transferable to AAA Games Development?}
+
+% Please do not change the subtitle
+\subtitle{COMP150 - Agile Development Practice}
+
+% Please put your student number in the author field
+\author{1606119}
+
+\begin{document}
+
+\maketitle
+
+\abstract{My agile development question revolves around one specific part of the Agile Development process and that is "User Stories", and how I intend to answer the aforementioned question is to use the knowledge we have gained from my research to ascertain whether the methods used to distribute them hold an effective place in games development, as after all they are an integral part of Agile Development practice as they are the medium of which developers are given their work in the development cycle to create a finished product, and then go further and look at the distribution methods such as Walking Skeleton and the MoSCow method, I will compare and contrast these individual methods, and by using my research gained from my noted sources, I will be able to ascertain if these methods have any effective place in the games development field, how they might by game developers, and if they are useful at all. And from these reflections on the different methods, I will be able to draw a final conclusion on my question and whether further research is required to gain a better understanding of the impact it can have on the usefulness of these methods in the video games industry.} 
+
+\section{Introduction}
+
+As the principles and philosophy of Agile Development is now common practice in both software and games development, the smaller parts of the overall "Agile" umbrella become more and more important to pick apart and evaluate to make sure that development goes as smoothly as possible and that "valuable software" and software is created "early and continuous" \cite{agilemanifesto}, so that projects are delivered to customers on time, which after all is the first point of the Agile Manifesto. And as User Stories are the way that work is presented to developers, their creation and distribution must be closely monitored to ensure that the scope of the project is not under or over evaluated, and that members of the team are not over encumbered with work or tasks they can't carry out efficiently, which would extend the length of a project; and there are several methods of User Story distribution that I have discovered through my research that are generally used in Agile Development, but I want to discover if these hold any place in the Games Development field and if they have the ability to aid development in anyway, and from my research, it seems that they don't have ability to.
```

Comment:
You can easily break down this paragraph into multiple points and ideas. This will also help make your writing more concise because, again, there is filler which doesn't communicate much to the reader. For example:
```
and there are several methods of User Story distribution that I have discovered through my research that are generally used in Agile Development, but I want to discover if these hold any place in the Games Development field
```
Could be refactored to:
```
Though several methods exist, it is unclear whether they are transferable to games development contexts.
```